src/bin/inx-chronicle/main.rs

```diff
@@ -16,16 +16,16 @@ mod stardust_inx;
 use bytesize::ByteSize;
 use chronicle::db::MongoDb;
 use clap::Parser;
+use config::ChronicleConfig;
 use tokio::task::JoinSet;
 use tracing::{debug, error, info};
-use tracing_subscriber::{fmt::format::FmtSpan, EnvFilter};
+use tracing_subscriber::{fmt::format::FmtSpan, prelude::*, EnvFilter};
```

Comment:
I really don't like preludes. How much things would we need to implement manually?